netlify.toml

```
@@ -430,3 +430,8 @@
   to = "/css/nixos-site.css"
   status = 302
   force = true
+
+[[headers]]
+  for "/.well-known/matrix/client"
```

cool, thanks!

```suggestion
  for = "/.well-known/matrix/client"
```


maybe? not sure, is the indentation right? the red highlighting is suspicious :)

---

Yeah, I was seeing that as well. My TOML-foo is worryingly non-existant.